actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -136,7 +136,7 @@ def expire_fragment(key, options = nil)
 
       def instrument_fragment_cache(name, key) # :nodoc:
         payload = instrument_payload(key)
-        ActiveSupport::Notifications.instrument("#{name}.#{instrument_name}", payload) { yield }
+        ActiveSupport::Notifications.instrument("#{name}.#{instrument_name}".freeze, payload) { yield }
```

Comment:
The `name` would either be `write_fragment`, `read_fragment`, `exist_fragment?` or `expire_fragment`, and `instrument_name` will always be the same. So it won't be dynamic.